examples/CMakeLists.txt

```diff
@@ -1,7 +1,7 @@
 # SPDX-License-Identifier: MIT
 
 add_executable(heat_equation heat_equation.cpp)
-target_link_libraries(heat_equation PUBLIC DDC::DDC)
+target_link_libraries(heat_equation PUBLIC DDC::DDC DDC::PDI_Wrapper)
```

Comment:
Shall we disable this example if `PDI_Wrapper` is not enabled ?

Reply:
True... Maybe `#ifdef` the PDI part

Reply:
Can you also add a if in the CMakeLists.txt